app/src/main/res/values-ro/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="IPP_info_version">Versiune</string>
+    <string name="IPP_info_package">Nume pachet</string>
```

Comment:
android uses the translation "Numele pachetului" in this [Url](https://support.google.com/admob/answer/9972781?hl=ro), which is the correct one?